[2025-01-17T14:36:49.915+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-17T14:36:49.940+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_data_to_duckdb.run_dbt_transform manual__2025-01-17T14:35:51.622601+00:00 [queued]>
[2025-01-17T14:36:49.952+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_data_to_duckdb.run_dbt_transform manual__2025-01-17T14:35:51.622601+00:00 [queued]>
[2025-01-17T14:36:49.954+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-17T14:36:49.983+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): run_dbt_transform> on 2025-01-17 14:35:51.622601+00:00
[2025-01-17T14:36:49.999+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1219) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-17T14:36:50.002+0000] {standard_task_runner.py:72} INFO - Started process 1224 to run task
[2025-01-17T14:36:50.001+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'user_data_to_duckdb', 'run_dbt_transform', 'manual__2025-01-17T14:35:51.622601+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/api_duckdb_dag.py', '--cfg-path', '/tmp/tmple93d2zi']
[2025-01-17T14:36:50.003+0000] {standard_task_runner.py:105} INFO - Job 164: Subtask run_dbt_transform
[2025-01-17T14:36:50.090+0000] {task_command.py:467} INFO - Running <TaskInstance: user_data_to_duckdb.run_dbt_transform manual__2025-01-17T14:35:51.622601+00:00 [running]> on host f3f45916ac6f
[2025-01-17T14:36:50.265+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='user_data_to_duckdb' AIRFLOW_CTX_TASK_ID='run_dbt_transform' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T14:35:51.622601+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-17T14:35:51.622601+00:00'
[2025-01-17T14:36:50.267+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-17T14:36:50.267+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-17T14:36:50.268+0000] {logging_mixin.py:190} INFO - Current task name:run_dbt_transform state:running start_date:2025-01-17 14:36:49.940971+00:00
[2025-01-17T14:36:50.269+0000] {logging_mixin.py:190} INFO - Dag name:user_data_to_duckdb and current dag run status:running
[2025-01-17T14:36:50.270+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-17T14:36:50.282+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-17T14:36:50.285+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        cd /opt/***/dags/dbt_models &&         dbt run --profiles-dir /opt/***/.dbt --target dev\n        ']
[2025-01-17T14:36:50.305+0000] {subprocess.py:99} INFO - Output:
[2025-01-17T14:37:00.318+0000] {subprocess.py:106} INFO - [0m14:37:00  Running with dbt=1.9.1
[2025-01-17T14:37:02.918+0000] {subprocess.py:106} INFO - [0m14:37:02  Registered adapter: duckdb=1.9.1
[2025-01-17T14:37:04.478+0000] {subprocess.py:106} INFO - [0m14:37:04  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-01-17T14:37:04.479+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-01-17T14:37:04.480+0000] {subprocess.py:106} INFO - - models.my_dbt_project
[2025-01-17T14:37:04.895+0000] {subprocess.py:106} INFO - [0m14:37:04  Found 424 macros
[2025-01-17T14:37:04.899+0000] {subprocess.py:106} INFO - [0m14:37:04  Nothing to do. Try checking your model configs and model specification args
[2025-01-17T14:37:06.574+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-17T14:37:06.682+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-17T14:37:06.684+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=user_data_to_duckdb, task_id=run_dbt_transform, run_id=manual__2025-01-17T14:35:51.622601+00:00, execution_date=20250117T143551, start_date=20250117T143649, end_date=20250117T143706
[2025-01-17T14:37:06.739+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-17T14:37:06.741+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-17T14:37:06.742+0000] {logging_mixin.py:190} INFO - Dag name:user_data_to_duckdb queued_at:2025-01-17 14:35:52.066538+00:00
[2025-01-17T14:37:06.744+0000] {logging_mixin.py:190} INFO - Task hostname:f3f45916ac6f operator:BashOperator
[2025-01-17T14:37:06.825+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-17T14:37:06.874+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-17T14:37:06.893+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
